substrate/client/rpc-spec-v2/src/chain_spec/api.rs

```diff
@@ -28,16 +28,16 @@ pub trait ChainSpecApi {
 	/// # Unstable
 	///
 	/// This method is unstable and subject to change in the future.
-	#[method(name = "chainSpec_unstable_chainName")]
-	fn chain_spec_unstable_chain_name(&self) -> RpcResult<String>;
+	#[method(name = "chainSpec_v1_chainName")]
+	fn chain_spec_v1_chain_name(&self) -> RpcResult<String>;
 
 	/// Get the chain's genesis hash.
 	///
 	/// # Unstable
 	///
 	/// This method is unstable and subject to change in the future.
```

Comment:
remove the unstable comment?

I regard it as "stabilized" after this PR

Reply:
Ops, forgot about those! Sure thing! Thanks